src/vs/editor/contrib/codeAction/browser/codeActionController.ts

```diff
@@ -335,3 +362,18 @@ export class CodeActionController extends Disposable implements IEditorContribut
 		return resultActions;
 	}
 }
+
+registerThemingParticipant((theme, collector) => {
+	const addBackgroundColorRule = (selector: string, color: Color | undefined): void => {
+		if (color) {
+			collector.addRule(`.monaco-editor ${selector} { background-color: ${color}; }`);
+		}
+	};
+
+	addBackgroundColorRule('.quickfix-edit-highlight', theme.getColor(editorFindMatchHighlight));
+	const findMatchHighlightBorder = theme.getColor(editorFindMatchHighlightBorder);
```

Comment:
Should this use a new theme color?

Reply:
regarding this and the above comment, using theming to closely follow what `matches` uses. 

The colors will still be tbd, so just using the matches highlighting for the time being